tests/benchmark/programs/aws_bucket/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "random_string" "bucket_name" {
+  length  = 8
+  special = false
+  upper   = false
+}
+
+resource "aws_s3_bucket" "example" {
+  bucket = random_string.bucket_name.result
```

Comment:
Probably a nit and a stretch - but a human converter would probably drop the random resource from the TF program and use pulumi auto-naming for the bucket. Is there a benchmark for "produces idiomatic Pulumi code" opportunity for the future?